Newspack/Newspack/Stores/SiteStore.swift

```diff
@@ -4,13 +4,25 @@ import WordPressFlux
 
 /// Responsible for managing site related things.
 ///
-class SiteStore: Store {
+class SiteStore: Store, FolderMaker {
 
-    private(set) var currentSiteID: UUID?
+    private(set) var currentSiteID: UUID? {
+        didSet {
+            // Note: didSet is never called during init.
```

Comment:
I'm **constantly** stumbling upon this issue as well. =(

Reply:
It's why I left myself a reminder! 😆